### ISSUE TITLE
New CLI: Review replacement for `move-rule`

### ISSUE THREAD

**Issue by dchristidis** [rucio/rucio#7265]
### Description

`rucio move-rule` effectively does two things:

1. Creates a new replication rule, copying most of the properties of an existing rule, except the RSE expression. Some may be explicitly overridden by the user.
2. Sets the child rule ID and expiration timestamp on the existing rule.

The current replacement (i.e. `rucio rule update --move`) makes sense in that it is like updating the RSE expression of an existing rule, but does so it two steps, changing the rule ID in the process. Still, out of the two actions, the creation of a new rule seems more important.

Some alternatives that may merit consideration:

1. If #536 were implemented, the moving a rule could be implemented as updating its RSE expression and re-evaluating it instead. (Personally, I think this offers the best experience but is the least likely to happen due to the complexity.)
2. Add an additional `rucio rule move` subcommand.

### Motivation

Here’s the help page for the `update` subcommand:

```console
$ rucio rule update --help
usage: rucio rule update [-h] [-a RULE_ACCOUNT] [--activity ACTIVITY] [--rule-id RULE_ID] [--lifetime LIFETIME]
                         [--locked] [--source-replica-expression SOURCE_REPLICA_EXPRESSION] [--comment COMMENT]
                         [--rses RSE_EXPRESSION] [--stuck] [--suspend] [--cancel-requests] [--priority PRIORITY]
                         [--child-rule-id CHILD_RULE_ID] [--boost-rule] [--move]

options:
  -h, --help            show this help message and exit
  -a, --account RULE_ACCOUNT
                        The account of the rule
  --activity ACTIVITY   Activity to be used (e.g. User, Data Consolidation)
  --rule-id RULE_ID     The rule ID, for accessing an existing rule.
  --lifetime LIFETIME   Rule lifetime (in seconds)
  --locked              Set the rule to locked
  --source-replica-expression SOURCE_REPLICA_EXPRESSION
                        RSE Expression for RSEs to be considered for source replicas
  --comment COMMENT     Comment about the replication rule
  --rses, --rse-exp RSE_EXPRESSION
                        RSE to change for the rule
  --stuck               Set state to STUCK
  --suspend             Set state to SUSPENDED
  --cancel-requests     Cancel requests when setting rules to stuck
  --priority PRIORITY   Priority of the requests of the rule
  --child-rule-id CHILD_RULE_ID
                        Child rule id of the rule. Use "None" to remove an existing parent/child relationship
  --boost-rule          Quickens the transition of a rule from STUCK to REPLICATING
  --move                Duplicate the existing replication rule to a different RSE
```

This is confusing in two ways:

1. It suggests you can use `--rse` without `--move`, e.g. `rucio update-rule --rule-id=… --rses=…`
2. It suggests you can use `--move` together with other options that are silently ignored, e.g. `rucio update-rule --move --rule-id=… --rses=… --comment=…`

**Comment by voetberg** [comment]
I'll vote for method 2 here, with the qualifier that if method 1 happens, `rucio rule move` would be depreciated. I agree with your assessment that method 1 is better but that it's unlikely for the issue to be addressed, and I will prefer doing something over nothing. 

Main thing I think the rewrite failed at was making clear what arguments work in which contexts, and this is a flagship example of that, defining `rule move` with very specific arguments will be an improvement.

**Comment by dchristidis** [comment]
Yes, I agree. The use of the five standardised verbs is nice, but adhering to this model when something doesn’t fit can be detrimental to the user experience.

**Comment by voetberg** [comment]
Closed by #7344
